demos/index.html

```diff
@@ -5,11 +5,11 @@
 		<meta charset="utf-8" />
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<link rel="stylesheet" href="https://build.origami.ft.com/v2/bundles/css?modules=o-typography@^4.3.2" />
+		<link rel="stylesheet" href="https://build.origami.ft.com/v2/bundles/css?modules=o-typography@^4.3.2,o-icons@^5.2.0,o-colors@^3.5.4" />
```

Comment:
I'm just thinking out loud but maybe we don't need to use the build service for our demos when we now have our build step and our bower components and our sass?